example-react-app/src/i18n/en.json

```diff
@@ -114,7 +114,13 @@
   "jmix.dataTable.operator._in": "in",
   "jmix.dataTable.operator._notIn": "not in",
   "jmix.dataTable.operator._isNull": "is empty",
-  "jmix.dataTable.operator.__inInterval": "in interval",
+  "jmix.dataTable.operator._inInterval": "in interval",
```

Comment:
This is not a mistake. https://github.com/Haulmont/jmix-frontend/blob/f90ce8563cd1ecdef691937c525dc15196cc283c/packages/jmix-react-core/src/crud/filter.ts#L1-L3